### DISCUSSION TITLE
How to change the field content server side when another field is changed

### DISCUSSION THREAD

**Discussion by Mte90** [Laravel-Backpack/community-forum#1223] [Q&A (Help)]
I have various fields define in the setup method but I need that one when the value is changed in another one updates the content a field.

[a comment by an automated account is not shown]

**Comment by karandatwani92** [comment]
Hey @Mte90 

Try this tutorial,  [How to access CRUD fields via javascript to manipulate Form](https://backpackforlaravel.com/articles/tutorials/how-to-access-crud-fields-via-javascript-to-manipulate-form)

**Reply by Mte90** [reply]
I did something similar but without the ajax call, just setting a url parameter and a refresh of the page.
